### PR TITLE
[Flutter Parent] Clean up usages of double quotes. Use l10n in domain search tests.

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020437&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/test/utils/accessibility_utils.dart [hunk 1]
```diff
@@ -97,10 +97,12 @@ class TextFieldNavigationGuideline extends AccessibilityGuideline {
   // Grab the focusable SemanticsNodes associated with this screen.
   List<SemanticsNode> _getFocusableSemanticsNodes(SemanticsNode root) {
     List<SemanticsNode> result = List<SemanticsNode>();
-    if(root.hasFlag(SemanticsFlag.isFocusable) && !root.isMergedIntoParent & !root.isInvisible && !root.hasFlag(SemanticsFlag.isHidden)) {
+    if (root.hasFlag(SemanticsFlag.isFocusable) &&
+        !root.isMergedIntoParent & !root.isInvisible &&
```
Comment (CalvinKern) [inline apps/flutter_parent/test/utils/accessibility_utils.dart]: Double checked with Joe. This should not be a bitwise and here, should be &&. Would you mind fixing this here?

Comment (JordanMarshall) [reply apps/flutter_parent/test/utils/accessibility_utils.dart]: 👍

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020440&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020441&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**